File: Calculate_Percentage/Percentage.java
/*
    Author : Pakshal Jain
    Date : 15/10/2022
    Program : To Calulate Precentage of Given Marks
*/

//Require Libraries
import java.lang.*;
import java.util.*;

//Class Declaration
class Calculate
{
    //Characteristics of Class
    int No = 0;
    float No2 = 0.0f;

    //Constructor of Class
    Calculate(int no,float no2)
    {
        this.No = no;
        this.No2 = no2;
    }

    //Behaviour of Class
    float Find()
    {
        //logic to calculate Percentage
        float ans = 0.0f;

        ans = this.No2 / this.No * 100;

        return ans;
    }
}

//Main Class
class Percentage 
{
    public static void main(String args[])
    {
        int No = 0;
        float No2 = 0.0f;
        float fret = 0.0f;

        //Creating object of Scanner Class To take input
        Scanner sobj = new Scanner(System.in);

        System.out.println("Jay Ganesh....");

        System.out.println("Enter Total Marks");
        No = sobj.nextInt();

        System.out.println("Enter Obtained Marks");
        No2 = sobj.nextFloat();

        //Creating object of Calculate Class
        Calculate cobj = new Calculate(No,No2);

        //Calling Behaviour of Find Class
        fret = cobj.Find();

        //Printing Final output
        System.out.println("Percentage is : "+fret+"%");
    }
}

File: Calculate_Percentage/Percentage.cpp
/*
    Author : Pakshal Jain
    Date : 15/10/2022
    Program : To Calculate Percentage
*/

//Required Libraries
#include<iostream>
using namespace std;
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

//Class Declaration
class Calculate
{
    //Chracteristics of Class
    private : 
        int No = 0;
        float No2 = 0.0;
        float ans = 0.0;
    //Constructor of Class
    public :
        Calculate(int no,float no2)
        {
            this->No = no;
            this->No2 = no2;
            this->ans = 0.0;
        }
    //Behaviour of Class
    float Percentage()
    {
        this->ans = this->No2 / this->No * 100;

        return this->ans;
    }
};

/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

int main() 
{
    int No = 0;
    float No2 = 0.0;
    float fret = 0.0;

    cout<<"Jay Ganesh....\n";
    cout<<"Enter Total Marks\n";
    cin>>No;

    cout<<"Enter Obtained Marks\n";
    cin>>No2;

    //Creating object of Class
    Calculate cobj(No,No2);

    //Calling Behaviour of Class
    fret = cobj.Percentage();

    printf("Percentage is : %.2f%%\n",fret);

    return 0;
}